tutorials/quantize.md

```diff
@@ -21,7 +21,9 @@ To reduce the memory requirements further, Lit-GPT supports several quantization
 
 ## `bnb.nf4`
 
-Enabled with [bitsandbyes](https://github.com/TimDettmers/bitsandbytes). Check out the [paper](https://arxiv.org/abs/2305.14314v1) to learn more about how it works.
+Enabled with [bitsandbytes](https://github.com/TimDettmers/bitsandbytes). Check out the [paper](https://arxiv.org/abs/2305.14314v1) to learn more about how it works.
```

Comment:
Thanks for fixing the  typo. Can you fix it in the other places of this file? Or the repo if you find any others

Reply:
Sure. I'll do it tomorrow.
What about trailing whitespaces, should I remove them too?

Reply:
Sure if you find any

Reply:
Done